app/views/shared/_api_selector.html.slim

```diff
@@ -1,11 +1 @@
-#api_selector
-
-javascript:
-  document.addEventListener('DOMContentLoaded', function () {
-    window.ContextSelector({
-      currentApi: #{json @service},
-      apis: #{json api_selector_services},
-      activeMenu: #{json active_menu},
-      audienceLink: #{json audience_link}
-    }, 'api_selector')
-  })
+#api_selector data={currentApi: @service, apis: api_selector_services, activeMenu: active_menu, audienceLink: audience_link}.to_json
```

Comment:
I think you should escape the json

Reply:
what's the difference with `.to_json`?

Reply:
I mean, it doesn't also escape js in the process?

Reply:
also: https://stackoverflow.com/questions/2715994/rails-escape-javascript-creates-invalid-json-by-escaping-single-quotes